dropwizard-servers/src/main/java/com/palantir/remoting1/servers/DropwizardTracingFilters.java

```diff
@@ -54,25 +54,26 @@ private DropwizardTracingFilters() {}
      * <p>
      * TODO(rfink) Is there a more stable way to retrieve IP/Port information?
      */
-    static void registerTracers(Environment environment, Configuration config, String tracerName) {
-        ServerTracer serverTracer = getServerTracer(extractIp(config), extractPort(config), tracerName);
+    static Brave registerTracers(Environment environment, Configuration config, String tracerName) {
```

Comment:
Do you have to plumb `Brave` through all the methods you want to trace locally? That seems to defeat the purpose of this setup. I'd be expecting a global static accessor.